src/uproot/_awkward_forth.py

```diff
@@ -53,7 +53,6 @@ def __init__(self, aform=None, count_obj=0, var_set=False):
         self.form_keys = []
         self.var_set = var_set
         self.count_obj = count_obj
-        return
```

Comment:
@aryan26roy and I are still refactoring this code, and unnecessary `return` statements like this would be on the chopping block, but thanks for catching it!